### PR TITLE
Adds MabaWebpackBundle

### PR DESCRIPTION
**Pull request by mariusbalcytis**

### CODE REVIEW

**Comment by stephanvierkant** [comment]
Why would I use this bundle instead of Symfony Encore?

**Comment by mariusbalcytis** [comment]
webpack-encore is just a wrapper for webpack inside JavaScript world. In other words, you could either use webpack manually, or use encore manually - the idea is quite the same, only the API of configuration is different.

What makes integration to Symfony easier is ability to use manifest file. It's provided by Symfony core and webpack plugin (not encore).

What this bundle adds is twig extension, which allows to automatically get all the entry points from your application. Either you're using plain webpack or encore, you have to collect your entry points manually and configure them separately in the config.

It also supports pointing entry point directly to image for it's optimisation and adding hash to the generated URL for cache busting.

In other words, it's more like AsseticBundle replacement (with webpack and it's goodies) while encore is easier configuration for webpack itself.

It's possible to use encore with this bundle, too, but the starting template is not prepared at this moment.

**Comment by jkufner** [comment]
This sounds more like a feature request to webpack-encore.

**Comment by mariusbalcytis** [comment]
Encore does not have any php code - it's just JavaScript wrapper on top of webpack to make commonly used configuration faster and use more-symfony-like naming for it. So, pull request is not an option - it does not have and won't have direct integration with twig.
